zetaclient/chains/ton/signer/signer.go

```diff
@@ -105,7 +108,13 @@ func (s *Signer) ProcessOutbound(
 
 	s.Logger().Std.Info().Fields(outbound.logFields).Msg("signing outbound")
 
-	if err = s.SignMessage(ctx, outbound.message, zetaHeight, nonce); err != nil {
+	if s.ClientMode == mode.DryMode {
+		s.Logger().Std.Info().Msg("dry-mode: skipping TON signing, sending, and tracking")
```

Comment:
I think we might want to use some standard to classify dry mode related logs so they can be easily filtered.
Like using a field mode == dry